components/next_steps_view/download_section.tsx

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import * as UserAgent from 'utils/user_agent';
```

Comment:
not sure if we actually want to import everything since we just use a couple of functions from these files?